.github/workflows/imagepuller-eval.yml

```diff
@@ -0,0 +1,48 @@
+name: imagepuller evaluate resource usage
```

Comment:
nit: maybe s/eval(uate)?/benchmark/ everywhere, imagepuller-eval sounds weird for nix ears.. :rabbit: